symbols/arm9.yml

```
@@ -4804,6 +4804,15 @@ arm9:
         NA: 0x2024428
         JP: 0x2024478
       description: "Note: unverified, ported from Irdkwia's notes"
+    - name: GetRankString
+      address:
+        EU: 0x2024D88
+      description: |-
+        Gets the string corresponding to the player's current explorer rank.
+        
+        r0: [output] str
```

Can you add more description? If this is the same as the later functions:

```suggestion
        r0: [output] Pointer to the buffer where the string will be written
```